src/main/kotlin/lotto/domain/LottoShop.kt

```diff
@@ -0,0 +1,34 @@
+package lotto.domain
+
+import lotto.domain.lotto.Lotto
+
+object LottoShop {
+
+    private const val LOTTO_PRICE = 1_000
+    private const val LOTTO_PRICE_ERROR_MESSAGE = "로또의 구매 단위는 1000 단위 입니다."
+    private const val ZERO = 0
+    private const val LACK_MONEY_MESSAGE = "금액이 부족 합니다."
+
+    private const val LOTTO_MIN_NUMBER = 1
+    private const val LOTTO_MAX_NUMBER = 45
+    private val LOTTO_RANGE = arrayOf(0, 6)
+
+    fun createLottoTicket(money: Int): List<Lotto> {
+        require(money % LOTTO_PRICE == 0) { IllegalArgumentException(LOTTO_PRICE_ERROR_MESSAGE) }
+        require(money > ZERO) { IllegalArgumentException(LACK_MONEY_MESSAGE) }
+
+        val lottoCount: Int = money / LOTTO_PRICE
+
+        val lottoList = mutableListOf<Lotto>()
+
+        for (i in 0..lottoCount) {
+            lottoList.add(Lotto(generatorLotto()))
+        }
+        return lottoList
```

Comment:
```suggestion
        return lottoList.toList()
```
방어적 복사를 활용해주시면 좋을 것 같아요

Reply:
bc4e3c7
위에 코멘트 repact와 방어적 복사 적용하였습니다.